content/chainguard/libraries/java/build-configuration.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Build Configuration"
+linktitle: "Build Configuration"
+type: "article"
+description: "Configuring Chainguard Libraries for Java on your workstation"
+draft: false
+tags: ["Libraries", "Java"]
```

Comment:
We have use "Chainguard Images" for tags previously, is "Chainguard Libraries" ok?

Reply:
Sure .. I am not sure yet about rules for tags..  made them consistent now